From 12ecfd11e326219ef185e7fdd6d967985fc57599 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20Br=C3=BCning?= <michael.bruning@qt.io>
Date: Thu, 20 May 2021 12:42:18 +0200
Subject: [PATCH] Bump V8_PATCH_LEVEL

Some backported patches to v8 require us to update the version string,
otherwise stale cache files might not be cleaned properly.

The V8_PATCH_LEVEL will de facto be decoupled from its upstream
counterpart, but that should not matter in practice.

Fixes: QTBUG-93744
Change-Id: I77d0d2a5b011955d22e6ae4c2eb45406f4c52863
Reviewed-by: Allan Sandfeld Jensen <allan.jensen@qt.io>
---
 chromium/v8/include/v8-version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chromium/v8/include/v8-version.h b/chromium/v8/include/v8-version.h
index cc87cda0776..298c9f74e02 100644
--- a/src/3rdparty/chromium/v8/include/v8-version.h
+++ b/src/3rdparty/chromium/v8/include/v8-version.h
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 8
 #define V8_MINOR_VERSION 7
 #define V8_BUILD_NUMBER 220
-#define V8_PATCH_LEVEL 31
+#define V8_PATCH_LEVEL 32
 
 // Use 1 for candidates and 0 otherwise.
 // (Boolean macro values are not supported by all preprocessors.)
